### ISSUE TITLE
Remove bullets before list checkbox items

### ISSUE THREAD

**Issue by qmarlats** [gogits/gogs#4249]
- Gogs version (or commit ref): 0.10.1.0228
- Can you reproduce the bug at https://try.gogs.io:
  - [x] [Yes](https://try.gogs.io/qmarlats/issues-tasks/issues/1)
  - [ ] No
  - [ ] Not relevant

## Description

I think there shouldn't be bullets next to checkboxes in tasks lists in issues. It should be more like Github. Also since you are using Semantic UI maybe you should consider using [their checkboxes](http://semantic-ui.com/modules/checkbox.html) (I don't know if it would render nice in issues, I didn't tested).

**Comment by Unknwon** [comment]
I think this is duplicated to https://github.com/gogits/gogs/issues/2049.

**Comment by qmarlats** [comment]
I don't think this is a duplicate, I was speaking about the UI. I think these bullets are ugly and useless (I call this "tasks lists" but I don't know if this is the right name):
![capture d ecran de 2017-03-06 23-11-34](https://cloud.githubusercontent.com/assets/3251811/23632633/dd9d26a6-02c2-11e7-8556-f775c2331612.png)
And I was also suggesting trying to use the checkboxes provided by Semantic UI in these tasks lists (if you put the checkboxes under a `<div class="ui checkbox">` they will get the Semantic UI design), but this is only a suggestion, I don't know how it could look.

**Comment by Unknwon** [comment]
Ah, I understand now.

OK, this is on purpose, it gains nothing for using Semantic UI's checkbox from my point of view. Current version is simple and just works.

**Comment by qmarlats** [comment]
Yep of course, however the bullets are really ugly ^^
